query/src/main/scala/filodb/query/exec/MetadataExecPlan.scala

```diff
@@ -395,75 +419,214 @@ final case class LabelCardinalityExec(queryContext: QueryContext,
     s" startMs=$startMs, endMs=$endMs"
 }
 
-final case object TopkCardExec {
+/**
+ * Contains utilities for all TsCardinality materialize() derivatives.
+ */
+final case object TsCardExec {
   // TODO: tune this
-  // See here for help choosing a size:
-  //   https://datasketches.apache.org/docs/Frequency/FrequentItemsErrorTable.html
-  val MAX_ITEMSKETCH_MAP_SIZE = 256;
+  val MAX_RESPONSE_SIZE = 5000
```

Comment:
This trips a `require` in `RowReader` that limits blob sizes to 64kb. Unsure yet how to bypass that...

Reply:
Resolved after schema change